examples/example1.nim

```diff
@@ -1,5 +1,5 @@
-include ../wxCompile
-#import wx
+#include ../wxCompile
+import ../wx
 
```

Comment:
If I get `example1.nim` correctly, It technically doesn't need `wx` at all, since it just emit C++ code directly. However, in order to compile correctly, it **does** need `wxCompile`